events/smart-descriptions.json

```
@@ -1,4 +1,337 @@
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "action.properties.start_tls",
+                    "value": "server"
+                }
+            ],
+            "relationships": [],
+            "value": "The server {action.properties.relay} sent {action.properties.bits_sent_ratio} bits with the verify status {action.properties.verify}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "event.reason",
+                    "value": "Sent"
+                }
+            ],
+            "relationships": [
+                {
+                    "source": "action.properties.relay",
+                    "target": "email.to",
+                    "type": "accepted a message form the following relay"
+                }
+            ],
+            "value": "{email.to} accepted a message form the following relay {action.properties.relay}"
+        },
+	     {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "event"
+                },
+                {
+                    "field": "even.reason",
+                    "value": "sent"
+                }
+            ],
+            "relationships": [],
+            "value": "A message to {email.to} was {action.properties.stat}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "spam"
+                }
+            ],
+            "relationships": [],
+            "value": "A spam activity has been detected from source IP {source.ip}: {event.message}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "virus"
+                },
+                {
+                    "field": "file.type",
+                    "value": "file"
+                },
+                {
+                    "field": "action.properties.event_status",
+                    "value": "CLEAN"
+                }
+            ],
+            "relationships": [],
+            "value": "A {file.type} nammed {file.name} with the related checksum: {file.hash.sha256} was concidered as {action.properties.event_status}"
+        },
+        {
+            "conditions": [
+                {
+                    "field": "event.kind",
+                    "value": "virus"
+                },
+                {
+                    "field": "file.type",
+                    "value": "file"
+                }
+            ],
+            "relationships": [],
+            "value": "A {file.type} nammed {file.name} with the related checksum: {file.hash.sha256} with no specific status"
```

```suggestion
            "value": "A {file.type} named {file.name} (SHA-256: {file.hash.sha256}) was scanned."
```

Not sure I understand this event correctly.